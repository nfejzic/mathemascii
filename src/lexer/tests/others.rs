use super::Snapshot;

super::test_snap!(roots, "sqrtroot");
super::test_snap!(calculus, "int oint del grad oo |...|");
super::test_snap!(shapes, "square diamond triangle /_\\ frown /_");
super::test_snap!(number_sets, "CCNNQQRRZZ");
super::test_snap!(floor_ceil_power, "^|__ __| |~ rceiling +-");
super::test_snap!(text, "text(some text) \"also text\"");
