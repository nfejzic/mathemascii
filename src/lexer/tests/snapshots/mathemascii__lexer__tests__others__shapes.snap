---
source: src/lexer/tests/others.rs
expression: "Snapshot((input, tokens))"
---
square diamond triangle /_\ frown /_

square
^^^^^^ -> Other(Square) at: 0 -> 6

       diamond
       ^^^^^^^ -> Other(Diamond) at: 7 -> 14

               triangle
               ^^^^^^^^ -> Other(Triangle) at: 15 -> 23

                        /_\
                        ^^^ -> Other(Triangle) at: 24 -> 27

                            frown
                            ^^^^^ -> Other(Frown) at: 28 -> 33

                                  /_
                                  ^^ -> Other(Angle) at: 34 -> 36


